cv_bridge/include/cv_bridge/cv_bridge.h

```diff
@@ -200,6 +222,47 @@ CvImageConstPtr toCvShare(const sensor_msgs::Image& source,
                           const std::string& encoding = std::string());
 
 /**
+ * \brief Convert a sensor_msgs::Image message to an OpenCV-compatible CvImage, copying the
```

Comment:
actually, no need to copy the doc, just declare it right before the non-compressed version